lib/sync_checker/formats/html_attachment_check.rb

```diff
@@ -98,8 +106,30 @@ def top_level_fields_hash
         }
       end
 
+      def attachment_should_exist_in_live_content_store?
+        Edition::PUBLICLY_VISIBLE_STATES.include?(attachable.state) ||
+          attachable_is_unpublished?
```

Comment:
Shouldn't this just be withdrawn, not any kind of unpublishing? If something's been consolidated or published in error, the attachments shouldn't be in the live content store.